src/model/controllers/task.controller.js

```diff
@@ -0,0 +1,107 @@
+const Task = require('../../models/task');
+
```

Comment:
src/model/controllers/task.controller.js

Your folder name is incorrect. Not model, but modules should be

Reply:
didn't fix